distribution/src/main/resources/licenses/notice.txt

```diff
@@ -37,8 +37,7 @@ commons-fileupload  http://jakarta.apache.org/commons/
 commons-httpclient  http://jakarta.apache.org/commons/ 
 commons-io  http://jakarta.apache.org/commons/ 
 commons-jxpath  http://jakarta.apache.org/commons/ 
-commons-lang    http://jakarta.apache.org/commons/ 
```

Comment:
Has this been intentionally removed? If not, it needs to stay not removed.

Reply:
Removed unintentional changes